packages/grid/x-data-grid-pro/src/hooks/features/rowReorder/useGridRowReorder.tsx

```diff
@@ -0,0 +1,145 @@
+import * as React from 'react';
```

Comment:
I've noticed that editing + row reordering combination acts weird:
- start cell editing
- reorder row
- end cell editing
- row is back at original position

https://user-images.githubusercontent.com/13808724/162024287-d7cf77f8-a58d-42a1-b858-194f9dc50fa6.mov

Codesandbox: https://codesandbox.io/s/roworderinggrid-material-demo-forked-ldorwe

Should editing be stopped once reordering starts? What do you think?

Reply:
I've disabled it, nice catch.